### DISCUSSION TITLE
Buttons Showing Up as Undefined

### DISCUSSION THREAD

**Discussion by jason-muljadi** [albertkun/21S-ASIAAM-191A#147] [Help Desk] [answered]
Hello! For the Lab 5 assignment, all my information has showed up as "undefined" for all the data. Yet, in my Live Server, everything shows up as is. How do I resolve that?

**Comment by jason-muljadi** [comment, marked as answer]
Never mind! It was just changing the permissions on Google Sheets and making sure that the object you're defining in init.js matches with the console.

**Reply by albertkun** [reply]
@jason-muljadi nice!! glad you were able to solve this!!
